Hyperbolic_surface_triangulation_2/doc/Hyperbolic_surface_triangulation_2/Concepts/ComplexNumber.h

```diff
@@ -0,0 +1,156 @@
+// Copyright (c) 2024 INRIA Nancy - Grand Est (France). LIGM Marne-la-Vallée (France)
+// All rights reserved.
+
+/*!
+\ingroup PkgHyperbolicSurfaceTriangulation2Concepts
+\cgalConcept
+
+Describes a complex number type that does not use square root.
+
+\cgalRefines{Field}
+
+\cgalHasModelsBegin
+\cgalHasModels{CGAL::Complex_number}
+\cgalHasModelsEnd
+*/
+class ComplexNumber {
+public:
+  /// \name Types
+  /// @{
+  /*!
+    Field number type: must be a model of `FieldNumberType`.
+  */
+  typedef unspecified_type FT;
+  /// \name Creation
+  /// @{
+  /*!
+    Default constructor, sets the both the real part and the imaginary part to \f$ 0 \f$.
+  */
+  ComplexNumber();
+
+  /*!
+    Constructor, sets the real part to <code> real_part </code> and the imaginary part to \f$ 0 \f$.
+  */
+  ComplexNumber(const FT& real_part);
+
+  /*!
+    Constructor, sets the real part to <code> real_part </code> and the imaginary part to <code> imaginary_part </code>.
+  */
+  ComplexNumber(const FT& real_part, const FT& imaginary_part);
+
+  /*!
+    Constructor, sets the real part to <code> real_part </code> and the imaginary part to <code> imaginary_part </code>.
+  */
+    template<class U,class V>
+      Complex_number(U&& real_part, V&& imaginary_part): _real(std::forward<U>(real_part)), _imag(std::forward<V>(imaginary_part));
+  /// @}
+
+  /// \name Get and Set
+  /// @{
+  /*!
+    sets the real part to <code> real_part </code>.
+  */
+  void real(const FT& real_part);
+
+  /*!
+    sets the imaginary part to <code> imaginary_part </code>.
+  */
+  void imag(const FT& imaginary_part);
+
+  /*!
+    returns the real part.
+  */
+  FT real() const;
+
+  /*!
+    returns the imaginary part.
+  */
+  FT imag() const;
+    /// @}
+
+  /// \name Operations
+  /// @{
+  /*!
+    returns the square of the modulus.
+  */
+  FT squared_modulus() const;
+
+  /*!
```

Comment:
There is no `std::complex<>::conjugate()`,  but `std::conj()`.   
I went to adapting your branch. Have a look [here](https://github.com/afabri/cgal/commit/5e3ab0a60a92f6c5d0f6eef22073d228cbfaa846).

Reply:
And similar for `squared_modulus()`